src/uwtools/config/formats/sh.py

```diff
@@ -27,17 +27,23 @@ def __init__(self, config: Union[dict, OptionalPath] = None):
 
     def _load(self, config_file: OptionalPath) -> dict:
         """
-        Reads and parses shell code consisting solely of key=value lines.
+        Reads and parses key=value lines from shell code.
```

Comment:
The current parsing technique should allow -- and ignore -- other valid shell code.